Jenkinsfile

```
@@ -157,7 +157,47 @@ pipeline {
                         }
                     }
                 }
+                stage ('Build Switch') {
+                    options {
+                        timeout(time: 20)
+                    }
+                    agent {
+                        label "SoH-Switch-Builders"
```

```suggestion
                        label "SoH-Linux-Builders"
```

this label property is looking for agents that are part of that group. This group does not exist. We should use a Linux machine to build this.